bot.py

```diff
@@ -628,6 +677,100 @@ async def show_prefix(self, message, lang, prefix):
+        e = self.tower_data.format_output_config(prefix=prefix, guild=message.guild, color=self.WHITE)
+        await self.answer(message, e)
+
+    async def set_tower_config_option(self, message, lang, prefix, option, value):
+        if self.is_guild_admin(message):
+            old_value, new_value = self.tower_data.set_option(guild=message.guild, option=option, value=value)
+
+            e = discord.Embed(title='Administrative action', color=self.RED)
+            e.add_field(name='Tower change accepted', value=f'Option {option} changed from `{old_value}` to `{new_value}`')
+            await self.answer(message, e)
+            log.debug(f'[{message.guild.name}] Changed Tower config {option} from {old_value} to {new_value}')
+        else:
+            e = discord.Embed(title='Administrative action', color=self.RED)
+            e.add_field(name='Tower change rejected', value=f'Only admins can change config options.')
+            await self.answer(message, e)
+
+    async def set_tower_config_alias(self, message, lang, prefix, category, field, values):
+        if self.is_guild_admin(message):
+            old_values, new_values = self.tower_data.set_alias(guild=message.guild, category=category, field=field, values=values)
+
+            e = discord.Embed(title='Administrative action', color=self.RED)
+            e.add_field(name='Tower change accepted', value=f'Alias {category}: {field} was changed from `{old_values}` to `{new_values}`')
+            await self.answer(message, e)
+            log.debug(f'[{message.guild.name}] Changed Tower config {category}: {field} from {old_values} to {new_values}')
+        else:
+            e = discord.Embed(title='Administrative action', color=self.RED)
+            e.add_field(name='Tower change rejected', value=f'Only admins can change config options.')
+            await self.answer(message, e)
+
+    async def show_tower_data(self, message, lang, prefix):
+        e = self.tower_data.format_output(guild=message.guild, channel=message.channel,
+            color=self.WHITE)
+        await self.answer(message, e)
+
+    async def edit_tower_single(self, message, lang, prefix, floor, room, scroll):
+        my_data = self.tower_data.get(message.guild)
+
+        short = my_data["short"]
+        
+        r = self.tower_data.edit_floor(prefix=prefix, guild=message.guild,
+            message=message, floor=floor, room=room, scroll=scroll)
+        
+        if short:
+            # Respond with a reaction.
+            await self.react(message, format_bool(r[0]))
+        else:
+            # Respond with an embed.
+            e = discord.Embed(title='Tower of Doom', color=self.WHITE)
+            e.add_field(name='Success' if r[0] else 'Failure', value=r[1])
+            await self.answer(message, e)
+            
+    async def edit_tower_floor(self, message, lang, prefix, floor, scrollA, scrollB, scrollC, scrollD, scrollE = None):
```

Comment:
please use snake_case, not camelCase (see PEP8)

Reply:
Fixed